class ObjectAlreadyExists(Exception):
    pass

class ValidationError(Exception):
    pass